application/common/models/EmailQueue.php

```diff
@@ -0,0 +1,219 @@
+<?php
+namespace common\models;
+
+use common\helpers\Utils;
+use common\models\User;
+use yii\helpers\ArrayHelper;
+use yii\helpers\Json;
+
+class EmailQueue extends EmailQueueBase
```

Comment:
Wacky suggestion, feel free to reject it: 
Would it make more sense to name these things `QueuedEmail`s? Overall, the table acts like an email queue, but that's not how table names end up being read. This isn't a table of email queues.

Our `User` table contains users, our `Method` table contains methods... so the table for queued emails would be `QueuedEmail`.  This just feels like it might make things more readable, because instance of models that represent rows in this table would now be named according to how we're actually using them.

Thoughts?  (Again, feel free to reject this or say we'll consider it some other time.)